nsls2forge_utils/all_feedstocks.py

```diff
@@ -0,0 +1,99 @@
+import requests
+import github3
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+def get_all_feedstocks_from_github(organization):
+    '''
+    Gets all public feedstock repository names from the nsls-ii-forge organization.
```

Comment:
Looks like it's a generic function. Maybe we can rephrase this line such as "... from a GitHub organization (e.g., nsls-ii-forge)?